### PR TITLE
add periodic job with nftables proxy

### PR DESCRIPTION
**Pull request by aojea**
/assign @danwinship

### CODE REVIEW

File: config/jobs/kubernetes/sig-network/sig-network-kind.yaml [hunk 1]
```diff
@@ -731,3 +731,51 @@ periodics:
     testgrid-tab-name: sig-network-kind, detect-local-interface-name-prefix
     description: Runs network tests using KIND against latest kubernetes master with a kubernetes-in-docker cluster and kube-proxy detectLocalMode=InterfaceNamePrefix
     testgrid-alert-email: antonio.ojea.garcia@gmail.com, widaly@microsoft.com
+- interval: 12h
```
Comment (danwinship) [inline config/jobs/kubernetes/sig-network/sig-network-kind.yaml:734]: ipvs is `6h`?

File: config/jobs/kubernetes/sig-network/sig-network-kind.yaml [hunk 2]
```diff
@@ -731,3 +731,51 @@ periodics:
...
+        limits:
+          cpu: 4
+          memory: 9Gi
+        requests:
+          cpu: 4
+          memory: 9Gi
+  annotations:
+    testgrid-dashboards: sig-network-kind, sig-testing-kind
+    testgrid-tab-name: sig-network-kind, nftables, master
+    description: Runs network tests using KIND against latest kubernetes master with a kubernetes-in-docker cluster
```
Comment (danwinship) [inline config/jobs/kubernetes/sig-network/sig-network-kind.yaml:779]: "and the nftables proxy mode"

File: config/jobs/kubernetes/sig-network/sig-network-kind.yaml (hunk 1 above)
Comment (aojea) [reply config/jobs/kubernetes/sig-network/sig-network-kind.yaml:782]: things from the past, I should update all of them to daily to reserve resources, in other areas without presubmits getting more granularity is good, but during the last years there was no benefit of running them on short cycles, we don't use to merge code in this area as often

File: config/jobs/kubernetes/sig-network/sig-network-kind.yaml [hunk 3]
```diff
@@ -100,6 +100,54 @@ presubmits:
...
+          limits:
+            cpu: 4
+            memory: 9Gi
+          requests:
+            cpu: 4
+            memory: 9Gi
+    annotations:
+      testgrid-dashboards: sig-network-kind
+      testgrid-tab-name: pr-sig-network-kind, nftables
+      description: Runs tests against a Kubernetes in Docker cluster suing ntables kube-proxy
```
Comment (danwinship) [inline config/jobs/kubernetes/sig-network/sig-network-kind.yaml:149]:
```suggestion
      description: Runs tests against a Kubernetes in Docker cluster using nftables kube-proxy
```

**Comment by aojea** [comment]
seems a flake

> > === Failed
> === FAIL: prow/entrypoint TestOptions_Run/interrupt,_propagate_child_error (0.30s)
> process started
> level=error msg="Entrypoint received interrupt: terminated"
> environment: line 5: kill: (31735) - No such process
> level=error msg="Process gracefully exited before 15s grace period"
> level=error msg="Error executing test process" error="process aborted"
>     run_test.go:255: interrupt, propagate child error: expected contents: "process started\nlevel=error msg=\"Entrypoint received interrupt: terminated\"\nlevel=error msg=\"Process gracefully exited before 15s grace period\"\n", got "process started\nlevel=error msg=\"Entrypoint received interrupt: terminated\"\nenvironment: line 5: kill: (31735) - No such process\nlevel=error msg=\"Process gracefully exited before 15s grace period\"\n"

/test pull-test-infra-unit-test

**Pull request merged**